test/spec/modules/realvuAnalyticsAdapter_spec.js

```diff
@@ -50,6 +50,7 @@ describe('RealVu', function() {
 
   describe('Analytics Adapter.', function () {
     it('enableAnalytics', function () {
+      this.timeout(5500)
```

Comment:
Could we remove changes to other adapter's spec files? If there are recommended changes I think we should go the route of letting the maintainer of those adapters know.